Disallow firing staff who are currently fixing or inspecting rides (#20260)
OpenRCT2/OpenRCT2: data/language/en-GB.txt

STR_6588    :Thanks to Herman Riddering for allowing us to record the 35er Voigt.
STR_6589    :Show window buttons on the left
STR_6590    :Show the window buttons (e.g. to close the window) on the left of the title bar instead of on the right.

> STR_6591    :Staff is currently fixing a ride and can't be fired.

Reviewer: ```suggestion
STR_6591    :Staff member is currently fixing a ride and can't be fired.
```

> STR_6591    :Staff member is currently fixing a ride and can't be fired.

Author: ```suggestion
STR_6591    :Staff member is currently fixing a ride and can’t be fired.
```

> STR_6592    :Staff is currently inspecting a ride and can't be fired.

Reviewer: ```suggestion
STR_6592    :Staff member is currently inspecting a ride and can't be fired.
```
